cfsdk/src/main/java/io/harness/cfsdk/cloud/Cloud.java

```diff
@@ -105,7 +105,7 @@ public boolean initialize() throws ApiException {
     public boolean isInitialized() {
 
         return this.authToken != null && this.authInfo != null &&
-                this.authInfo.getEnvironmentIdentifier() != null;
```

Comment:
This would have resulted in `dev` , `prod` etc. and not the actual UUID of the environment. The relay proxy doesn't send this and the SDK wouldn't auth.

Reply:
@andybharness I decoded to keep the isInitialised check based on the UUID